README.md

```diff
@@ -28,20 +22,42 @@ If you use chor-js in an academic setting, please cite our demo paper:
 > [[Bibtex]](https://dblp.org/rec/bibtex/conf/er/LadleifWW19)
 
 ## Installation
-chor-js is packaged [via npm](https://www.npmjs.com/package/chor-js).
 
-## Usage
+### a) Pre-Packaged
 
-Create a chor-js instance and link it to a canvas:
+Just include the pre-packaged code in your webpage:
+
+```html
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/chor-js@latest/dist/assets/chor-js.css">
+<script src="https://cdn.jsdelivr.net/npm/chor-js@latest/dist/chor-js-modeler.min.js"></script>
+<!-- ... or 'viewer' or 'navigated-viewer'! -->
+```
+
+You can find a sample webpage [here](./docs/prepackaged.html).
+
+### b) NPM
+
+Install the package via `npm install chor-js` and import chor-js in your application:
 
 ```javascript
-// Import modeler or viewer class
 import ChorJS from 'chor-js/lib/Modeler';
```

Comment:
Hier muss es im übrigen nicht ChorJS sein, der name ist frei wählbar, ChoreoModeler wäre also auch möglich

Reply:
True, but this way it fits the example later on, so I will leave it this way.